sql/item_mods.sql

```diff
@@ -43267,7 +43275,7 @@ INSERT INTO `item_mods` VALUES (27676,23,17); -- Attack+17
 INSERT INTO `item_mods` VALUES (27676,31,29); -- Magic Evasion+29
 INSERT INTO `item_mods` VALUES (27676,384,600); -- Haste+6%
 INSERT INTO `item_mods` VALUES (27676,5,19); -- MP+19
-INSERT INTO `item_mods` VALUES (27676,65,15); -- Evasion+15
+INSERT INTO `item_mods` VALUES (27676,108,15); -- Evasion+15
```

Comment:
Modifier 108 is Evasion _skill_ and modifier 68 is actual evasion. You want 68.

![image](https://user-images.githubusercontent.com/6871475/97951743-3604fc00-1d69-11eb-983e-678d6ec3b970.png)